spec/person_spec.rb

```diff
@@ -0,0 +1,9 @@
+require './lib/person.rb'
+require 'date'
+require 'yaml'
```

Comment:
Can't see a need for this require at this point. Same thing for `date` Don't require moduls that you don't use.